include/fuse_common.h

```diff
@@ -32,14 +32,22 @@ extern "C" {
 #endif
 
 /**
- * Information about open files
+ * Information about an open file handle
+ *
+ * File Handles are created by the open, opendir, and create methods and closed
+ * by the release and releasedir methods.  Multiple file handles may be
+ * concurrently open for the same file.  Generally, a client will create one
+ * file handle per file descriptor, though in some cases multiple file
+ * descriptors can share a single file handle.
```

Comment:
Can we omit the last sentence please? I think this is confusing - people will be uncertain what "client" means in this context, and the file handle isn't actually created by the client but may be created by the filesystem.

Reply:
How about "client will open" instead of "will create"?

Reply:
I think that's still more confusing than helpful... Either people already know how the kernel maps file descriptors to FUSE file handles (in which case they don't need this information), or they don't know (in which case they won't understand it from this sentence either).

Reply:
I think that most libfuse users would prefer their file systems to be portable, so they need to know what the least common denominiator client will do.  From the users' perspective, they need clear rules on they're allowed to do with libfuse.  Is there something I should add here to make it clearer?